app/models/user-password.server.ts

```diff
@@ -13,9 +13,29 @@ export const query = {
 }
 
 export const mutation = {
-  async update({ id, password }: z.infer<typeof schemaUserUpdatePassword>) {
+  async update({
+    id,
+    password,
+    currentPassword,
+  }: z.infer<typeof schemaUserUpdatePassword>) {
     const hashedPassword = await bcrypt.hash(password, 10)
```

Comment:
```suggestion
    const hashedPassword = await bcrypt.hash(password, 10)
```

I think you can be move in line *38*.

Reply:
Hi @r17x , I'm Fikri and Nice to meet you too. I'm new comer contributor too and this is my first pull request, thank you so much for the feedback.